sources/targets/Stride.Core.targets

```diff
@@ -46,10 +46,7 @@
         Condition="'$(StrideEnableCodeAnalysis)' != 'true'">
     <ItemGroup>
       <!-- We want to include Stride analyzers by default -->
-      <AnalyzersNotToRemove Include="@(Analyzer)" Condition="$([System.String]::Copy(%(FullPath)).Contains('Stride'))" />
-      <!-- We want to include source generators by default -->
-      <AnalyzersNotToRemove Include="@(Analyzer)" Condition="$([System.String]::Copy(%(FullPath)).Contains('Generator'))" />
-      <AnalyzersNotToRemove Include="@(Analyzer)" Condition="$([System.String]::Copy(%(FullPath)).Contains('SourceGeneration'))" />
+      <AnalyzersNotToRemove Include="@(Analyzer)" Condition="$(FullPath.Contains('Stride'))" />
```

Comment:
This shouldn't be part of that PR. It has nothing to do with translation.

Reply:
Yep, my mistake. Please don't include it on the PR.

Reply:
you will have to push a commit reverting this change or revert it with git tools

Reply:
Ill keep this stalled until we see the new Avalonia Port.